velox/expression/CastExpr-inl.h

```diff
@@ -206,6 +206,35 @@ void CastExpr::applyIntToDecimalCastKernel(
       });
 }
 
+template <typename TOutput>
```

Comment:
Is this template parameter needed? Isn't it always int128_t?

Reply:
It could be int64_t.

Reply:
Got it. Thank you for clarifying. It would be nice to add a comment explaining that.